scripts/set_assignees.py

```diff
@@ -76,10 +76,6 @@ def process_pr(gh, maintainer_file, number):
             manifest_change = True
             break
 
-    # one liner PRs should be trivial
-    if pr.commits == 1 and (pr.additions <= 1 and pr.deletions <= 1) and not manifest_change:
```

Comment:
there's a compliance test for unused variable as well

Reply:
Fixed.